ncbi_genome_download/__main__.py

```diff
@@ -9,21 +9,21 @@
 def main():
     """Build and parse command line"""
     parser = argparse.ArgumentParser()
-    parser.add_argument('domain',
-                        choices=dflt.DOMAIN.choices,
-                        default=dflt.DOMAIN.default,
-                        help='The NCBI "domain" to download (default: %(default)s)')
+    parser.add_argument('group',
```

Comment:
NCBI calls this "domain", so I'd like to stick to that term, even if it's wrong.